packages/flutter_tools/test/general.shard/resident_runner_test.dart

```diff
@@ -2681,14 +2516,12 @@ void main() {
     DevtoolsLauncher: () => mockDevtoolsLauncher,
   });
 
-  testUsingContext('HotRunner unforwards device ports', () => testbed.run(() async {
+  testUsingContext('HotRunner calls device dispose', () => testbed.run(() async {
```

Comment:
This test was testing that a mock it setup got called by another mock it setup. I re-wrote it so it only checks that the method called by the ResidentRunner is invoked correctly instead of testing the (mocked) implementation of Device.dispose.

Reply:
Gotta make sure the mockito library works!